integration-tests/pom.xml

```diff
@@ -194,6 +194,119 @@
 			</build>
 		</profile>
 
+		<profile>
```

Comment:
update copyright notice

Reply:
Thanks for reviewing this for me. I updated all the files according to your review. Can you review it again? Thanks, -Xian